.circleci/config.yml

```diff
@@ -99,6 +102,27 @@ jobs:
           path:  build/distributions
           destination: distributions
           when: always
+  testWindows:
+    executor: win/default
+    steps:
+      - attach_workspace:
+          at: ~/project
+      - run:
+          name: Unzip Windows build
+          no_output_timeout: 20m
+          command: |
+            cd build/distributions
+            unzip besu-*.zip -d besu-tmp
+            cd besu-tmp
+            mv besu-* ../besu
+      - run:
+          name: Test Besu Windows executable
+          no_output_timeout: 10m
+          command: |
+            build\distributions\besu\bin\besu.bat --help
+            build\distributions\besu\bin\besu.bat --version
```

Comment:
What does this do before the fix?  Does the orb expect a zero from the process return?

Reply:
That's right. Before the fix, the process exits with code 1, and therefore the build fails.

This is a very simple sanity check.